packages/cli/lib/projects.js

```diff
@@ -4,10 +4,15 @@ const path = require('path');
 const chalk = require('chalk');
 const findup = require('findup-sync');
 const { prompt } = require('inquirer');
+const Spinnies = require('spinnies');
```

Comment:
We were using `ora` for loading spinners elsewhere in the CLI (example https://github.com/HubSpot/hubspot-cli/blob/9da73e88b7c0d4d37993f1cc3a6a53d588c8d36f/packages/cli/commands/logs.js#L78). Is there a reason to have both or should we use `ora` here also?

Reply:
Yeah, that's exactly it and I agree